packages/web3torrent/src/clients/payment-channel-client.ts

```diff
@@ -1,11 +1,12 @@
 import {ChannelResult, Message, ChannelClientInterface} from '@statechannels/channel-client';
-import {bigNumberify} from 'ethers/utils';
+import {utils} from 'ethers';
```

Comment:
Why?

Reply:
It allowed storybook to not throw errors , see https://github.com/ethers-io/ethers.js/issues/349